projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -7,16 +7,18 @@
   "scripts": {
     "cy:open": "cypress open",
     "cy:open:b2b": "cypress open --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers",
+    "cy:open:epd-visualization": "cypress open --env BASE_SITE=powertools-epdvisualization-spa",
     "cy:run": "cypress run",
     "cy:run:b2b": "cypress run --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers",
     "cy:run:ci": "cypress run --config-file $CYPRESS_CONFIG --record --key $CYPRESS_KEY --tag \"2011,b2c,all,parallel\" --parallel --group B2C --ci-build-id $TRAVIS_BUILD_ID --spec \"cypress/integration/!(vendor|b2b|ssr)/**/*.e2e-spec.ts\"",
     "cy:run:ci:ssr": "cypress run --config-file $CYPRESS_CONFIG --config baseUrl=http://localhost:4000 --record --key $CYPRESS_KEY --tag \"2011,ssr,all,parallel\" --parallel --group SSR --ci-build-id $TRAVIS_BUILD_ID --spec \"cypress/integration/ssr/pages.e2e-spec.ts\"",
     "cy:run:ci:b2b": "cypress run --config-file $CYPRESS_CONFIG --record --key $CYPRESS_KEY --tag \"2011,b2b,all\" --group B2B --ci-build-id $TRAVIS_BUILD_ID --spec \"cypress/integration/b2b/**/*.e2e-spec.ts\"",
-    "cy:run:ci:cds": "cypress run --config-file $CYPRESS_CONFIG --record --key $CYPRESS_KEY --tag \"2011,b2c,all-cds\" --spec \"cypress/integration/!(vendor|b2b)/**/*.e2e-spec.ts,cypress/integration/vendor/cds/**/*.e2e-spec.ts\"",
+    "cy:run:ci:cds": "cypress run --config-file $CYPRESS_CONFIG --record --key $CYPRESS_KEY --tag \"2011,b2c,all-cds\" --group CDS --spec \"cypress/integration/vendor/cds/**/*.e2e-spec.ts\"",
```

Comment:
Is this change really needed in this PR? If not, let's move it into a separate PR.

Note: I couldn't find this change in the PR to develop.

Reply:
This is not an intended change. I will remove it.